expression/aggregation/descriptor.go

```diff
@@ -110,6 +110,7 @@ func (a *AggFuncDesc) Clone() *AggFuncDesc {
 	for i, byItem := range a.OrderByItems {
 		clone.OrderByItems[i] = byItem.Clone()
 	}
+	clone.Mode = a.Mode
```

Comment:
dose previous function like Equal() and window functions need consider Agg mode?